hack/test/e2e-run.sh

```diff
@@ -13,8 +13,8 @@ export DOCKER_ENGINE_GOARCH=${DOCKER_ENGINE_GOARCH:-${ARCH}}
 : ${TESTDEBUG:=}
 
 integration_api_dirs=${TEST_INTEGRATION_DIR:-"$(
-	find ./integration -type d |
-	grep -vE '(^./integration($|/internal)|/testdata)')"}
+	find /tests/integration -type d |
```

Comment:
Bit confused by this one; this forces the directory to be at the root?

Reply:
yeah that's how the `e2e` image is set-up.. I could change that though :sweat_smile:

Reply:
Yeah, I think it should be flexible, and not rely on a fixed path (ideally), perhaps either an env-var, or relative to where the script is run from?